Vagrantfile

```diff
@@ -3,25 +3,60 @@
 
 Vagrant.configure("2") do |config|
 
-  config.vm.define 'grsec-build' do |build|
-    build.vm.box = "trusty64"
+  config.vm.define 'grsec-build', primary: true do |build|
+    build.vm.box = "ubuntu/trusty64"
     build.vm.hostname = "grsec-build"
-    build.vm.box_url = "https://cloud-images.ubuntu.com/vagrant/trusty/current/trusty-server-cloudimg-amd64-vagrant-disk1.box"
 
     build.vm.provision :ansible do |ansible|
-      ansible.playbook = 'ansible/build-deb-pkg.yml'
-      ansible.verbose = 'v'
-      # The grsec build playbook defaults to patching
-      # and building the vanilla kernel. Building for
-      # Ubuntu hosts requires a few tricks, so enable them.
+      # Two playbooks are available for building the kernel, for test and stable patches.
+      # The stable playbook will prompt for HTTP auth credentials prior to running.
+      # One subsequent runs of the stable playbook, you can simply hit enter to skip
+      # the credential entry, since the files have already been downloaded.
+      ansible.playbook = 'ansible/build-grsecurity-kernel-test.yml'
+      ansible.playbook = 'ansible/build-grsecurity-kernel-stable.yml'
+      ansible.verbose = 'vv'
+      # Exposing the build strategy var for testing various build strategies quickly.
       ansible.extra_vars = {
-        'grsecurity_build_strategy' => 'ubuntu',
+        'grsecurity_build_strategy' => 'vanilla',
       }
     end
-
     build.vm.provider "virtualbox" do |v|
-      v.name = "grsec-build"
       v.memory = 2048
+      v.customize ["modifyvm", :id, "--cpus", available_vcpus]
     end
   end
+
+  # Separate machine for testing installation of .deb packages.
+  # In case of problems, you don't want to pollute the build machine
+  # with the test packages.
+  config.vm.define 'grsec-install', autostart: false do |install|
+    install.vm.box = "ubuntu/trusty64"
+    install.vm.hostname = "grsec-install"
+    # If grsec install works, the shared folder mount will fail.
+    # Uncomment the line below to disable it.
+    # install.vm.synced_folder './', '/vagrant', disabled: true
+  end
+end
+
+
+def available_vcpus
+  # Increase number of virtual CPUs in guest VM.
+  # Rather than blindly set it to "2" or similar,
+  # inspect the number of VCPUs on the host and use that,
```

Comment:
Good that the hard-coding of the mem usage isn't a problem. Figured if you're compiling kernels and don't have 2GB of RAM to spare, a simple edit of the Vagrantfile isn't too much to ask.